app/cruds/cruds_raffle.py

```diff
@@ -38,14 +39,14 @@ async def create_raffle(
         raise err
 
 
-async def get_raffle_by_groupid(
+async def get_raffles_by_groupid(
     group_id: str,
     db: AsyncSession,
-) -> models_raffle.Raffle | None:
+) -> list[models_raffle.Raffle] | None:
```

Comment:
With `all()` method, the crud will return an empty list instead of None
```suggestion
) -> list[models_raffle.Raffle]:
```